CSS/tablet.css

```diff
@@ -141,3 +168,105 @@
   margin-left: auto;
   margin-right: auto;
 }
+
+/* popup window */
+#popup-window {
+  padding: 3rem 4.5rem;
+  background-color: rgba(193, 199, 208, 0.7);
+  backdrop-filter: blur(9px);
+}
+
+#popup-container {
+  display: flex;
+  flex-direction: column;
+}
+
+#popup-window i.fa-xmark {
+  position: fixed;
+  top: 4.2rem;
+  right: 6rem;
+  font-size: 21px;
+  color: grey;
+}
+
+#popup-window .buttons {
+  display: flex;
+  flex-direction: row;
+  gap: 1rem;
+  align-items: center;
+  justify-content: center;
+}
+
+#popup-window .buttons a {
+  width: fit-content;
+  font-size: 15px;
+}
+
+#popup-window .card {
+  gap: 0;
+}
+
+#popup-window .card img {
+  width: 100%;
+  height: 50%;
+}
+
+#popup-window .card .card-content {
+  width: 100%;
+  gap: 0;
+}
+
+@media screen and (min-width: 900px) {
+  #popup-container {
+    padding: 0.9rem;
+  }
+
+  #popup-window .card img {
+    height: 100%;
+  }
+
+  #popup-window .card-details {
+    display: flex;
+    flex-direction: row;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 30px;
+    color: #344563;
+  }
+
```

Comment:
- [ ] You have done a great a job on this milestone but there are littl things you need to amend to complet milestone .
- [ ] The languages & button are besides the text  ⚠️not appear as the figma design, 👁️ 
- [ ] The `see live button` and `see source button`  button not appear as the figma design
i would kindly suggest you to fix all this  so as to make your page look profeesional. 
- [ ] your  design 
![Screenshot 2023-05-28 015817](https://github.com/Roathy/-Portfolio-/assets/92458236/95a6eaf0-b79b-4cc1-be8c-6c6d225c0338)

- [ ] Figma design
![Screenshot 2023-05-28 012943](https://github.com/Roathy/-Portfolio-/assets/92458236/6661ff85-0007-4f62-bf0e-8beacaba2fc9)


- [ ] your design on desktop 
![Screenshot 2023-05-28 015709](https://github.com/Roathy/-Portfolio-/assets/92458236/bf7fae09-fb86-402a-be5f-08990ff7f49e)

- [ ] figma design on desktop
![Screenshot 2023-05-28 012811](https://github.com/Roathy/-Portfolio-/assets/92458236/246ac981-a4be-47a8-85af-7662f4b74681)